packages/office-ui-fabric-react/src/components/Divider/VerticalDivider.styles.ts

```diff
@@ -0,0 +1,41 @@
+import { IVerticalDividerPropsStyles, IVerticalDividerStyles } from './VerticalDivider.types';
+import { IStyleFunction } from '../../Utilities';
+
+export const getStyles: IStyleFunction<IVerticalDividerPropsStyles, IVerticalDividerStyles> = (
+  props: IVerticalDividerPropsStyles
+): IVerticalDividerStyles => {
+  const { theme, getClassNames, className } = props;
+
+  if (!theme) {
```

Comment:
maybe throw an error if the theme is not reaching styles to notify that something is wrong instead of just providing empty styles?